### ISSUE TITLE
Default to 3 standard deviations

### ISSUE THREAD

**Issue by AlistairNWard** [chmille4/bam.iobio.io#87]
When bam.iobio is opened, set the default in the read coverage chart to less standard deviations (maybe 3 or 4)

**Comment by anderspitman** [comment]
@AlistairNWard, the vue branch is currently defaulting to 4 standard deviations. Do we want to change it to 3?

**Comment by anderspitman** [comment]
Ok now I'm confused. I tried it with a different data set and now it says "multiples of median" instead of standard deviations. Which part of the UI are we talking about, the "Zoom y axis" widget?

**Comment by AlistairNWard** [comment]
Yes, we're talking about the "Zoom y axis" widget. Should have been clearer. I also wonder if the slider is the best element to control this as it is very laggy. This might do better as just having the number of standard deviations listed and the user can just change the value? This isn't responsive enough to move the slider and watch its effect.

And yes, this was actually implemented as multiples of the median coverage.

The purpose of this is to try to find the median coverage and assume that this represents the coverage of standard diploid genome. A heterozygous deletion would then appear at half the median coverage and duplications/CNVs would have coverage increases in multiples of half of the median, if this makes sense. There are lots of big coverage spikes, and this chart was automatically scaling to accommodate all data, which meant you couldn't really pick out what was happening around the median coverage. So if we set the max on the y axis to a few multiples of the median coverage, we should be able to pick out large CNVs. I would really like to see a way of smoothing this data to make it more legible, but that is another issue!
